backend/src/v5/models/jobs.js

```diff
@@ -19,8 +19,24 @@ const Jobs = {};
 
 const DBHandler = require('../handler/db');
 
+const DEFAULT_JOBS = [
+	{ _id: 'Client', color: '#a6cee3' },
```

Comment:
we should fix https://github.com/3drepo/3drepo.io/issues/2150 whilst we're here.

Can we add an extra default job, `Admin` to this list

And at init, we should assign he teamspace owner to this job.

We will also need a migration script to create this role and assign the owner if the owner does not currently have a job